### ISSUE TITLE
[Question] Combine MSI and MSIX

### ISSUE THREAD

**Issue by TopperDEL** [oleg-shilo/wixsharp#1194]
I'm not sure if this is the right place to ask. If not, please let me know.

I want to install a WinUI-3-app where I can simply create a MSIX from within Visual Studio. But I need to install an MSI upfront so I think I need a Bootstrapper here.
But: the Wixsharp-Bootstrapper can only include MSI - or the other way: I do not know how to install an MSIX from a Wixsharp bootstrapper.

My second idea then was to create an MSIX from my existing MSI and bundle them together, but the bootstrapper also only generates a MSI.

Basically I want to bring an EXE or MSI-Installer to the Windows Store, that installs "Dokany" (virtual file system driver) using existing MSI from Dokany before my WinUI-App gets installed. But I'm completely lost currently. Any help is therefore greatly appreciated. 
Thank you!

**Comment by oleg-shilo** [comment]
WixSharp is all about simplifying using WiX toolset. Thus, if WiX lacks some core technology support, WixSharp will only have a limited opportunity to help. 
Thus WiX does not support MSIX at least out of the box (it's somewhat available in the non-free close source WiX Expansion Pack).

This also applies to the Bootstrapper support. As far as I know, WiX Bootstrapper has no specific support for MSIX packages (though I can be wrong). 
However, you can always use any deployable packages in a generic way. Something like this pseudocode:

```xml
<Chain>
        <ExePackage Name="PseudoMsix" ... 
            exe="msiexec" 
            installargs=$"/i {msix from the bundle payload}" 
            uninstallargs=$"/u {msix from the bundle payload}"... 
</Chain>
```

**Comment by TopperDEL** [comment]
Ok, thank you!
